pkg/liqoctl/version/handler.go

```diff
@@ -0,0 +1,32 @@
+// Copyright 2019-2021 The Liqo Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package version
+
+import (
+	"fmt"
+)
+
+// if you need to rename the following variables, change also the build command for liqoctl.
+var liqoCtlVersion = "development"
+var liqoCtlSHA = ""
```

Comment:
It seems to me this is not configured by the pipeline.

Reply:
Yes, I talked with @palexster and we decided to keep it there and not include it in the pipeline because at the moment the pipeline doesn't differentiate between the version and the commit ref